lib/client/email/fb-email-client.js

```diff
@@ -34,32 +34,24 @@ class FBEmailClient extends FBJWTClient {
    * @param {object} payload
    * Email data
    *
-   * @param {object} [logger]
-   * Bunyan logger instance
-   *
    * @param {object} [options]
    * Email send options
    *
+   * @param {object} [options.logger]
+   * Bunyan logger instance
+   *
    * @return {promise<undefined>}
    *
    **/
-  async sendMessage (payload, logger, options = {}) {
+  async sendMessage (payload, options = {}) {
     const url = '/email'
 
     payload.service_slug = this.serviceSlug
 
-    try {
-      const response = await this.sendPost({
-        url,
-        payload
-      }, logger)
-      return response
-    } catch (e) {
-      if (options.throwOnError) {
-        throw e
-      }
-      return {}
-    }
+    return this.sendPost({
+      url,
+      payload
+    }, options.logger)
```

Comment:
Interested to know why the logger sits on options now.  Is it required to complete this save & return functionality?

Reply:
I thought it was simpler, that's all.